app/messages/package_update_message.rb

```diff
@@ -2,7 +2,8 @@
 
 module VCAP::CloudController
   class PackageUpdateMessage < MetadataBaseMessage
-    register_allowed_keys []
+    register_allowed_keys %i[username password]
+    attr_reader :package
```

Comment:
Is this needed?

Reply:
I meant the second line (`attr_reader :package`). I don't see `package` being used in this file.